lib/pulsar/client/config_util.py

```diff
@@ -58,9 +58,6 @@ def __find_type(path, explicit_type, default_type):
 
```

Comment:
You had a wrong merge with `dev` here, you are missing this: https://github.com/natefoo/galaxy/commit/033b1e88da99b86ffbd33321374aa59d1d72484c#diff-8ce0f1f93264f339e0bf364c7c19d3bdL4